packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -298,6 +298,239 @@ curl -v -X GET \
+
+##### Request Body
+| Property | Type                     | Description                          |
+| -------- | -------------------------|--------------------------------------|
+| `policyTypes` | Array| (Optional) Supported PolicyTypes for simulation: OKTA_SIGN_ON, MFA_ENROLL, PROFILE_ENROLLMENT, ACCESS_POLICY. Default `null`, return all types|
+| `appInstance` | String | (Required) The appInstance ID for this simulation |
+| `policyContext.user.id` | String | (Required) The userId for this simulation. Only userId or groupIds allowed, not both  |
+| `policyContext.groups.ids` | Array | (Required) The groupIds for this simulation. Only userId or groupIds allowed, not both |
+| `policyContext.ip` | String | (Optional) The network rule condition, zone or IP address. See [Network Condition](#network-condition-object) |
+| `policyContext.zones.ids` | String | (Optional) The zone ID under the network rule condition. See [Network Condition](#network-condition-object) |
+| `policyContext.risk` | String | (Optional) The risk rule condition: LOW, MEDIUM, HIGH. See [Risk Score Condition](#risk-score-condition-object)|
+| `policyContext.device.managed` | Boolean | (Optional) If the device is registered. See [Device Condition](#device-condition-object)|
+| `policyContext.device.registered` | Boolean | (Optional) If the device is managed. See [Device Condition](#device-condition-object)|
+| `policyContext.device.platform` |String | (Optional) The platform of the device, eg: IOS. See [Platform Condition](#platform-condition-object)|
+| `policyContext.device.assuranceId` | String | (Optional) The device assurance policy ID for the simulation|
+
+##### Request example
+> **Note:** You can only evaluate `user` or `groups` not both, same for the `ip` and `zone.ids`. Use payload below for reference only
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '
+{
+    "policyTypes": ["OKTA_SIGN_ON", "MFA_ENROLL"],
+    "appInstance": "0oa4eroj3nYCIJIW70g7",
+    "policyContext": {
+        "user": {
+            "id": "00u4eralyizp8NPOb0g7"
+        },
+        "groups": {
+            "ids": [
+              "00g4eralvekR5RLuS0g7", "00g4eralvekR5RLuS0g8"
+            ]
+        },
+        "risk": {
+            "level": "LOW"
+        },
+        "ip": "9.9.9.9",
+        "zones": {
+          "ids": [
+            "nzo4eralxcRnbIHYJ0g7"
+            ]
+        },
+        "device": {
+            "platform": "IOS",
+            "registered": true,
+            "managed": true
+        }
+    }
+}' "https://${yourOktaDomain}/api/v1/policies/simulate?expand=EVALUATED&expand=RULE"
+```
+
+#### Response
+The response section below explains the error responses and the response body
+
+##### Error Responses
```

Comment:
Do we need to include error responses? Don't see it for other APIs on this page

Reply:
I agree. I think we can leave these out.